index.js

```diff
@@ -12,6 +12,63 @@ function query(query, format = "CSV") {
   return chdbNode.Query(query, format);
 }
 
+// Standalone exported query function
+function queryBuffer(query, format = "CSV") {
+  if (!query) {
+    return "";
+  }
+  return chdbNode.QueryBuffer(query, format);
```

Comment:
the chdbNode.QueryBuffer seems not exist

Reply:
:+1:  Your right, I've not cleaned up this code :(